data/human/post-war reactions.txt

```diff
@@ -0,0 +1,257 @@
+				goto next
+			label me
+			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You led the attack! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
+				goto next
+			label blew
+			`	"I bet you did! I wish I was on one of our warships, bravely defending us all. Blowing up Pugs, getting paid. It sounds amazing!"`
+			label next
+			`	"What was it like, fighting those nasty, evil Pugs?"`
+
+			choice
+				`	"Actually, they're not evil. Their 'war' was a ploy to get us to stop fighting among ourselves. A lot of Pug gave their lives willingly to save us from self-destruction."`
+				`	"War is hell, kid. I'm not going to glorify it for you."`
+					goto hell
+				`	"Leading a battle is like chess with guns and engines. It's the most exhilarating feeling I've ever felt."`
+					goto chess
+				`	"It was an honor to protect the human race from our attackers."`
+					goto honor
+			`	"Woah, you sound really smart. Like one of those sensible experts in the vids! Still, without people like you, the Pug's plan couldn't have worked. I still reckon you were the best, Captain."`
+				goto salute
+			label hell
+			`	"You went through hell for us, eh? That sounds less fun than I thought... Maybe I should stick to school and be an engineer instead of a pilot. I dunno."`
+				goto salute
+			label chess
+			`	"The Syndicate is training me to maintain and they're even going to train me to pilot their combat ships when I'm older! One day, I'll fight aliens like you did."`
+				goto salute
+			label honor
+			`	"Honor, and service. The Syndicate will train boys like me, but I've wondered about joining the Navy. It isn't as profitable, but it sounds more noble."`
+			label salute
+			`	You arrive at the repair shop, and an older girl, perhaps seventeen, meets the two of you. Gus stands to attention and gives you a well-practiced salute. You politely salute him in return.`
+			`	As Gus ends his salute, the girl attacks him from behind, putting him into a choke hold. It's clear she's taking extreme care when doing it. "Did you wander out of the childcare district? Was it the spaceport landing area this time? I can see scratches and bruises all over you." Gus replies, "I'm sorry, Sally! I was curious."`
+			`	The girl lets him out of the hold and says to you, "Thank you for saving my brother. He's the only family I have left, and I hope," she pinches him, "he'll stop risking his life out of curiosity." She looks at Gus kindly. "Come on, let's get you to the medic."`
+			`	Sally waves goodbye as she walks her brother into the repair shop orphanage.`
```

Comment:
I think this could optionally just simply be truncated with the new context.
```suggestion
			`	Sally waves goodbye as she walks her brother in.`
```